docs/api/createSlice.md

```diff
@@ -107,6 +115,8 @@ for references in a larger codebase.
 import { createSlice } from '@reduxjs/toolkit'
 import { createStore, combineReducers } from 'redux'
 
+const incrementBy = createAction < number > 'incrementBy'
```

Comment:
ummmh... wtf is prettier doing here...?

Reply:
@markerikson I can't convince prettier of `const incrementBy = createAction<number>('incrementBy')` here...
Do you face the same problem?

Reply:
The code block is marked as `js`, not `ts`, so it's being mis-formatted.

And given that we aren't using any other TS in there, the example is kinda inconsistent anyway.